user_guide_src/source/installation/running.rst

```diff
@@ -426,6 +426,85 @@ Setting Environment
 
 See :ref:`Handling Multiple Environments <environment-nginx>`.
 
+
+.. _deployment-to-shared-hosting-services:
+
+*************************************
+Deployment to Shared Hosting Services
+*************************************
+
+.. important::
+    **index.php** is no longer in the root of the project! It has been moved inside
+    the **public** folder, for better security and separation of components.
+
+    This means that you should configure your web server to "point" to your project's
+    **public** folder, and not to the project root.
+
+Specifying the Document Root
+============================
+
+The best way is to set the document root to the **public** folder in the server
+configuration::
+
+    └── example.com/ (project folder)
+        └── public/  (document root)
+
+Check with your hosting service provider to see if you can change the document root.
+Unfortunately, if you cannot change the document root, go to the next way.
+
+Using Two Directories
```

Comment:
Well, I tested this method in practice, it works well.
@kenjis, thank you for following up.

Reply:
This method (and a few hacks) worked on 000webhostapp.
https://kenjis.000webhostapp.com/

Reply:
Oh!! In 2022, someone asked me this question in an email :innocent: 

![Screenshot 2024-02-20 131713](https://github.com/codeigniter4/CodeIgniter4/assets/9530214/c5865a98-4da3-4f9e-9c02-7d81432b1d4b)